Frontend/src/App.jsx

```diff
@@ -45,6 +49,9 @@ function App() {
           <Route path="/contact" render={() => <Contact setUser={setUser} />} />
           <Route path="/password_reset" render={() => <PasswordResetPage />} />
           <Route path="/result" render={() => <VisualizationPage />} />
+          <Route path="/alex" render={() => <AtlasModelChoice />} />
+          <Route path="/explore" render={() => <Explore />} />
+          <Route path="/genemapper" render={() => <UploadFilePage />} />
```

Comment:
This is not where the page is supposed to be in the future, but we can leave it in for now for testing purposes.

Reply:
Yes exactly, right now it is just for development, will make sure not to remove it later on.